crypto/encode_decode/decoder_pkey.c

```diff
@@ -314,7 +314,8 @@ int ossl_decoder_ctx_setup_for_pkey(OSSL_DECODER_CTX *ctx,
 
     if (keytype != NULL
             && (strcmp(keytype, "id-ecPublicKey") == 0
-                || strcmp(keytype, "1.2.840.10045.2.1") == 0))
+                || strcmp(keytype, "1.2.840.10045.2.1") == 0
+                || strcmp(keytype, "ECC") == 0))
```

Comment:
Should it be EC instead? Where is the ECC coming from?

Reply:
It should be EC. I have changed it to EC.